appserver/jdbc/jdbc-config/src/main/java/org/glassfish/jdbc/config/JdbcConnectionPool.java

```diff
@@ -190,14 +187,12 @@
     /**
      * Gets the value of the maxPoolSize property.
      *
-     * Maximum number of conections that can be created
+     * Maximum number of connections that can be created
      * 
      * @return possible object is
      *         {@link String }
      */
     @Attribute (defaultValue="32")
-    @Min(value=1)
-    @Max(value=Integer.MAX_VALUE)
     String getMaxPoolSize();
```

Comment:
Isn't removing the min/max value begging for illegal values to be configured? Shouldn't they be at least checked for in code somewhere?

Reply:
I agree it is a rather risky move, although as far as I'm aware these annotations will always cause a validation exception in the event that a user tries to input a string that cannot be parsed to an Integer (as would be the case with a system variable name) I've altered the basic check in the implementation of JdbcConnectionPool so as to prevent the value from being below 1 but I do agree that it isn't the safest situation

Reply:
I think @svendiedrichsen has a good point here. Surely extending the mechanism isn't part of the task as originally requested but if we don't include this or make it a precondition of this PR we introduce room for error that might never be closed again even when we would like to see this happen in the near future.